amanuensis/resources/userdatamodel/userdatamodel_project.py

```diff
@@ -101,7 +101,9 @@ def create_project(current_session, user_id, description, name, institution, sea
     current_session.flush()
     new_project.searches.extend(searches)
     new_project.requests.extend(requests)
-    new_project.associated_users.extend(associated_users)
+    role_id = current_session.query(AssociatedUserRoles.id).filter(AssociatedUserRoles.code == "METADATA_ACCESS").first()[0]
```

Comment:
Handle errors like you did in the other file when role_id is null because METADATA_ACCESS is not present in the DB.

Reply:
That error appear when a user tried input a role that didnt exist. if no role was given if would default to Metadata_access but under the assumption that it exists in the db. should I just not add any associated users to the project if metadata is missing?

Reply:
I just added the error handler so the associated users wouldn't be added to the project if METADATA_ACCESS is not in the DB